lms/djangoapps/survey/tests/factories.py

```diff
@@ -0,0 +1,21 @@
+# pylint:disable=missing-docstring
+import factory
```

Comment:
Just to be clear, you're not using SurveyFormFactory/SurveyAnswerFactory and CreditRequirementFactory/CreditRequirementStatusFactory in this PR, right? Those are both P2 stuff that I'm going to use in my ticket for the second endpoint?

Reply:
Yes, thank you, I was about to add that to the PR description.

Reply:
Gotcha, ok. Just making sure because I think I copied these factories into my PR, too, because I wasn't sure if you were going to keep them in this one. It's fine if you keep them though, I'll just rebase.